### DISCUSSION TITLE
Foxglove.dev "Download" only shows partial list of available topics

### DISCUSSION THREAD

**Discussion by valschmidt** [foxglove/community#1015] [Bugs]
When I select a time period in Foxglove.dev and then attempt to download a file with only selected topics in it, (this interface)...
![image](https://github.com/user-attachments/assets/b3830b0e-8e70-43a7-9ceb-cbff166fda20)
I get only a partial list of the topics that are actualy available at that time in the record. A search only returns results from the topics shown, so this does not help either. 

This feel like a bug to me.

**Comment by defunctzombie** [comment]
Hi - could you email support [at] foxglove [dot] dev with this description. Please also include your org name/slug and a link to the timeline URL (or note the start/end time of the data you are trying to download) so we can look into the describe behavior.
